build/dependencies.props

```diff
@@ -5,40 +5,40 @@
   <!-- This files is typically managed by automation. Execute 'run.ps1 upgrade deps' to update these variables to the last-known-good versions. -->
   <PropertyGroup Label="Package Versions">
     <BenchmarkDotNetPackageVersion>0.10.9</BenchmarkDotNetPackageVersion>
-    <InternalAspNetCoreSdkPackageVersion>2.1.0-preview1-15576</InternalAspNetCoreSdkPackageVersion>
+    <InternalAspNetCoreSdkPackageVersion>2.1.0-preview1-15620</InternalAspNetCoreSdkPackageVersion>
```

Comment:
Hypothetically this shouldn't break anything, but I try to keep the version in ./korebuild.lock.txt the same as this version. You can update that version by running `.\run.ps1 -Update install-tools`.